_pages/en/employee-profiles.md

```diff
@@ -1,23 +0,0 @@
----
-layout: default
-title: Employee Profiles
-ref: employee-profiles
-lang: en
-status: posted
-sections: Work In Progress
-permalink: /employee-profiles.html
----
-
-## Employee Profiles
-
-ESDC is a large department (~28 000 employees) of the GC that includes 2 portfolio organizations, the [Labour Program](https://www.canada.ca/en/employment-social-development/corporate/portfolio/labour.html) and [Service Canada](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada.html).
-
-These profiles are to help categorize employees and determine their IT and business needs when creating a [long term](it-picture-long-term.md) and [medium term](it-picture-medium-term.md) visions as well as strategies to get there.
-
-### Profiles
-
-- Management - Manager and above
-- Power users - Development, system/database administration, analytics/reporting
-- Client services - Work directly with the public
-- General users - Usually at their desk
-- Special needs - Non-standard devices and tools
```

Comment:
This page is for personas of ESDC employees.  Not a profile page for the team

Reply:
This page has been removed and can now be found on PR #512

Reply:
#512 modifies the file, but it should not be removed from in this PR.  It will cause a conflict.

Reply:
Hmm, will have a look. #512 is off the master directly so yeah, I guess I should just leave it as is. Pretty sur it'll still generate a conflict in the end though.